conf.d/60-geoip.conf

```diff
@@ -0,0 +1,3 @@
+[options]
+# This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com
```

Comment:
Actually we don't include data, so it's better not to lie IMHO.

```suggestion
```

Reply:
Fixed it. Its also better not to lie in my opinion. Thanks for noticing the error.